task2/func.js

```diff
@@ -0,0 +1,23 @@
+function NamedOne(firstName, lastName) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.fullName = function (newValue) {
+        if (newValue) {
+            const array = newValue.split(' ');
+            if (array.length === 2) {
+                this.firstName = array[0];
+                this.lastName = array[1];
+            }
+        }
+        return `${this.firstName} ${this.lastName}`;
+    }
+}
+
+var	namedOne = new NamedOne("Naomi","Wang");
```

Comment:
I understand that this part of the code was copied, but, please, don't use var and also redundant spaces:
just `const namedOne = new NamedOne("Naomi","Wang");`